design_docs/openapi.yml

```diff
@@ -6,166 +6,229 @@ info:
     name: TBD
     url: TBD
 servers:
-  - url: http://TBD.mongodb.com/v1/ # URL TBD
+  - url: http://TBD.mongodb.com/v1
+security:
+  - ApiKeyAuth: []
 paths:
-  /start:
+  /conversations:
     post:
-      operationId: start
-      summary: Start a new conversation with the chatbot
-      description: |
-        This endpoint is used to start a new conversation with the chatbot.
-        The response contains metadata for the conversation.
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              properties:
-                user_ip:
-                  type: string
-                  format: ipv4
-                  description: The IP address of the user performing the conversation.
-              required: [user_ip]
+      operationId: createConversation
+      summary: Start a new conversation
       responses:
-        "200":
-          description: |
-            Successful operation. Respond with text.
-            Response format depends on `stream` query parameter.
+        200:
+          description: OK
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  conversation_id:
-                    type: string
-                    description: Unique identifier for the conversation
-                required: [conversation_id]
-        "400":
-          description: Invalid input
-        "401":
-          description: Unauthorized
-        "500":
-          description: Internal server error
-      security:
-        - CORS: []
-  /respond:
+                $ref: '#/components/schemas/Conversation'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        404:
+          $ref: '#/components/responses/NotFound'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+  /conversations/{conversationId}:
```

Comment:
i don't think we need these endpoints for MVP. i'm ok keeping in spec because at some point it'd probably be good to add, but pls mark w a comment saying out of MVP scope